### PR TITLE
adjust dependencies to handle opensearch

### PR DESCRIPTION
**Pull request by MJedr**

### CODE REVIEW

File: .github/workflows/build-and-release.yml [hunk 1]
```diff
@@ -9,43 +9,39 @@ on:
...
       - name: Checkout
         uses: actions/checkout@v2
         with:
           fetch-depth: 0
 
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.8
+          architecture: x64
```
Comment (drjova) [inline .github/workflows/build-and-release.yml:22]: do we have to specify it?

File: run-tests.sh [hunk 2]
```diff
@@ -22,4 +22,4 @@
 
 set -e
 
-pytest --pep8 -vvv tests
+py.test tests
```
Comment (drjova) [inline run-tests.sh:25]: why we need to remove the pep8 check?

Comment (MJedr) [reply run-tests.sh:25]: it's deprecated

File: setup.py [hunk 3]
```diff
@@ -52,7 +51,6 @@
     'flake8-future-import~=0.0,>=0.4.3',
     'mock~=2.0,>=2.0.0',
     'pytest-cov~=2.0,<2.6.0',
-    'pytest~=3.0,>=3.6.0',
```
Comment (drjova) [inline setup.py:55]: we should leave the pytest dep

**Pull request merged**